src/components/Form/Form.vue

```diff
@@ -20,7 +20,7 @@
         .field(v-else-if="Object.keys(item).includes('slot')",
               v-bind="item.attr",
               data-test="slot")
-          slot(:name="Object.values(item)[0]")
+          slot(:name="item['slot']", v-bind="item.props")
```

Comment:
@14nrv I would like to also add a `ValidationProvider`, but I cannot get it to work, any idea?

I took your Control.vue as a reference and tried the following: 

```pug
.field(v-else-if="Object.keys(item).includes('slot')",
              v-bind="item.attr",
              data-test="slot")
          ValidationProvider(
            tag="div"
            :vid="item.props.vid"
            :rules="getRules(item.props)"
            :name="item.props.name || item.props.label | slugify"
            :immediate="!!item.props.value"
            v-slot="{ errors }")
            slot(:name="item['slot']", v-bind="item.props")
            p.help.is-danger(v-if="errors.length") {{ errors[0] }}
```

And the validation won't pass even if I have set the value in `item.props.value` via `item.props. updateValue()`.

Any idea?

Reply:
No idea for now, its need "deep" investigation IMO